Refactor emscripten_get_now() implementation to avoid Closure warnings.

juj opened the pull request:
In particular this does a change
```js
var _emscripten_get_now = performance.now;
```
as opposed to
```js
var _emscripten_get_now = function() { return performance.now(); };
```
That drops `this==performance` in invocations, and I really vaguely recall that some environment would not have liked that, but cannot remember which, and if we don't have a documented proof, we shouldn't do it.

(compare to Date.now, which was never invoked with `this==Date`)

kripken commented on src/library.js:
```diff
@@ -4047,24 +4046,25 @@ LibraryManager.library = {
 #else
                                // Modern environment where performance.now() is supported:
-                               "_emscripten_get_now = function() { return performance['now'](); };\n",
+                               "return performance['now'];\n" +
```
do we still need quotations for `performance.now`? maybe with newer closure that is already recognized.

juj commented:
Hey @kripken, given that I changed the way that `emscripten_get_now()` is written out, I needed to also lean on the same scheme as `emscripten_memcpy_big` does. That scheme did not work for `emscripten_get_now()` as-is, so I changed that to a `xxx__import: true` directive that can be used for both. Can you take a second look on this PR?
